### PR TITLE
RFE: header updates

### PR DESCRIPTION
**Pull request by tych0**
There's a bunch of stuff that has been added in recent kernels, let's add it here.

### CODE REVIEW

**Comment by tych0** [comment]
Oh, but I guess we should define them in include/seccomp.h.in too, let me do that.

File: src/system.h [hunk 1]
```diff
@@ -61,6 +61,7 @@ struct db_filter_col;
 #define SECCOMP_RET_TRAP	0x00030000U /* disallow and force a SIGSYS */
 #define SECCOMP_RET_ERRNO	0x00050000U /* returns an errno */
 #define SECCOMP_RET_TRACE	0x7ff00000U /* pass to a tracer or disallow */
+#define SECCOMP_RET_LOG		0x7ffc0000U /* allow after logging */
```
Comment (pcmoore) [inline src/system.h]: We added SECCOMP_RET_LOG back in 3b22b153c70ef00044fb309d9a9fecc12b777e98, does that commit not work for you?

Comment (pcmoore) [reply src/system.h]: I suppose we could add this bit of the patch, but drop the section below as commit 3b22b153 should have that covered.

**Comment by pcmoore** [comment]
Thanks.

I merged the first two patches via 635496a5938a57bd0b3877f59504b7e7d07b37c0 and b09e6e800646de6f5cbee4efeb02c5b1ee4e09cf.  I'm going to leave the SPEC_ALLOW changes for after the v2.4 release.

File: src/system.h (hunk 1 above)
Comment (tych0) [reply src/system.h]: This duality between system.h and seccomp.h.in is a little bit overengineered, IMO.

However, I'm going to import SECCOMP_RET_USER_NOTIF into system.h (for testing that the return type is supported) in the user-notification series, so I think it makes sense to add this one here as well, to keep things in sync with the kernel headers. Although yes, it's a no-op for this series.

**Comment by pcmoore** [comment]
Since the SPEC_ALLOW stuff is part of the user notification PR let's close this out.  @tych0 if you feel otherwise feel free to comment and we can reopen this PR.

**Pull request closed without merging**